Packs/Slack/Integrations/SlackEventCollector/SlackEventCollector.py

```diff
@@ -132,23 +139,29 @@ def get_logs_with_pagination(self, query_params: dict, last_run: dict) -> list[d
             '''
             If didn't fetch logs, we are not changing the last run
             If fetched logs, There are 4 scenarios
-                1. This run we did a new query and finished fetching all the events, so we save the newest event as 'last_fetched_event'.
-                2. We did a new query this time and did not finish fetching all the events, so we save the newest event as 'newest_event_fetched' and the 'cursor' for the next run.
-                3. We continued to fetch events by 'cursor' and finished fetching them all, saving the 'newest_event_fetched' as 'last_fetched_event'.
-                3. We continued to fetch events by 'cursor', and we still haven't finished fetching them all, so we only need to save the 'cursor'.
+                1. This run we did a new query and finished fetching all the events,
+                    so we save the newest event as 'last_fetched_event'.
+                2. We did a new query this time and did not finish fetching all the events, 
+                    so we save the newest event as 'newest_event_fetched' and the 'cursor' for the next run.
+                3. We continued to fetch events by 'cursor' and finished fetching them all, 
+                    saving the 'newest_event_fetched' as 'last_fetched_event'.
+                3. We continued to fetch events by 'cursor', and we still haven't finished fetching them all, 
+                    so we only need to save the 'cursor'.
```

Comment:
In #2 and #4 we also save the `last_search_stop_point_event_id` no?

Reply:
Yes, I added it